docs/sources/developers/plugins/extend-the-grafana-ui-with-links.md

```diff
@@ -71,20 +82,20 @@ new AppPlugin().configureExtensionLink({
 });
 ```
 
-The above example demonstrates how to return a different link `path` based on which plugin the dashboard panel is using. If the clicked-upon panel is neither a timeseries nor a piechart panel, then the configure function returns `undefined` and the link isn't rendered.
+The above example demonstrates how to return a different `path` based on which plugin the dashboard panel is using. If the clicked-upon panel is neither a time series nor a pie chart panel, then the `configureExtensionLink` function returns _undefined_. When this happens, Grafana doesn't render the link.
 
-_Note: The context passed to the `configure` function is bound by the `extensionPointId` the link is inserted into. Different extension points contain different contexts._
+> **Note**: The context passed to the `configureExtensionLink` function is bound by the `extensionPointId` into which you insert the link. Different extension points contain different contexts.
 
-## Add an onClick event handler to a link
+## Add an event handler to a link
 
-Link extensions provide the means to direct users to a plugin page via href links within the Grafana UI. They can also trigger onClick events to perform dynamic actions when clicked.
+Link extensions give you the means to direct users to a plugin page via href links within the Grafana UI. You can also use them to trigger `onClick` events to perform dynamic actions when clicked.
 
-Here's how you can add an onClick handler to a link in the dashboard panel menus of Grafana via your plugin:
+Your plugin can add an `onClick` event handler to a link in Grafana's dashboard panel menus:
```

Comment:
We need an intro to the steps, like "To add an event handler to a link, complete the following steps:"

Reply:
Done.